tutorials/time-series-modeling/Lab_05.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Time Series Modeling
+description: This lab will go over the fundamentals of Time Series Modeling, including decomposition, autocorrelation and modeling historical changes.
+author: ghidora77, benamie
+tags: remote-sensing, time-series, satellite
+date_published: 2022-02-01
+---
+<!--
+Copyright 2021 The Google Earth Engine Community Authors
```

Comment:
Change year to 2022
(We need to update our template(!))

Reply:
Updated